### ISSUE TITLE
ZeroProxy Multi-user support

### ISSUE THREAD

**Issue by shakna-israel** [HelloZeroNet/ZeroBlog#2]
Probably more for [the proxy project](https://github.com/zeronetproxy/ZeroNetProxy), but users can edit the site, though not sign it, and pollute the proxy.

E.g. A user repeatedly hits the "Add post" button, (as seen on (http://zero.network:43110/1Bu6sb3PYD1XzbcqqQgVw42S6C83Yzdweb at the moment).

It DOES NOT affect the ZeroNet apart from the proxy, because it isn't signed or published.
It DOES affect anyone deciding to use that proxy.
It DOES affect everyone using this blog style on that proxy.

**Comment by HelloZeroNet** [comment]
Yeah, ZeroNet is not ready for this situation yet. Next version will feature BIP32 based site auth support, and it makes multi-user feature possible.
